code-of-conduct.md

```diff
@@ -0,0 +1,78 @@
+
+Everyone who participates in list-related activities is required to conform to this Code of Conduct. It applies to all spaces related to the Expert Volunteer List, including, but not limited to, the private interactions between expert volunteers and scientists seeking support, the list entry submission form, the project's GitHub repository, list-related email, and list-related discussion in other public forums. By participating, participants indicate their acceptance of the procedures by which the list's maintainers resolve any Code of Conduct incidents, which may include storage and processing of their list-related information. 
+
+##### Part 2.1. Expected behaviour
+
+All participants in list-related activities are expected to show respect and courtesy to others. All interactions should be professional regardless of platform. In order to foster a positive and professional  environment we encourage the following kinds of behaviours in interactions:
+
+* Use welcoming and inclusive language
+* Be respectful of different viewpoints and experiences
+* Gracefully accept constructive criticism
+* Show courtesy and respect towards others
+
+Experts should also be mindful of the fact that scientists seeking support may be living through (or have recently lived through) extremely traumatic experiences. Be considerate and avoid bringing up topics that may cause anxiety. Conversely, be prepared for the possibility of hearing accounts that could be disturbing. We recommend focusing on the science and if necessary setting clear boundaries on other topics.
+
+##### Part 2.2. Unacceptable behaviour
+
+Examples of unacceptable behaviour by participants in any list-related activites include:
+
+- written or verbal comments which have the effect of excluding people on the basis of membership of any specific group  
+- causing someone to fear for their safety, such as through stalking, following, or intimidation  
+- violent threats or language directed against another person
+- the display of sexual or violent images  
+- unwelcome sexual attention or contact of any form
+- insults or put downs  
+- sexist, racist, homophobic, transphobic, ableist, or exclusionary jokes
+- excessive swearing
+- incitement to violence, suicide, or self-harm  
+- continuing to initiate interaction with someone after being asked to stop  
+- publication of private communication without consent  
+
+##### Part 2.3. Consequences of Unacceptable behaviour
+
+Participants who are asked to stop any inappropriate behaviour are expected to comply immediately. This applies to all list-related activities on any platform. If an expert volunteer or scientist seeking support engages in behaviour that violates this Code of Conduct, their counterpart may warn the offender, terminate the interaction, and/or inform the list's maintainers via email to [expertvolunteerlist@gmail.com](expertvolunteerlist@gmail.com). 
```

Comment:
Good question. It sounds like it might be a good thing to do, but I can see quite a few problems with it:

1. Is it ok to inform a whole list of people about someone's alleged misbehavior without having been able to verify the allegations in any way? Broadcasting someone's details seems like a very different beast compared to "silently" removing their details from a list (as we will do for experts). Then again, they accept the CoC by participating, so they know this could happen.
2. Even if it's ok in principle, is it ok to keep this information online as a public record in some sort of "offender list"? After how much time would people's details be removed from such a list?
3. If a permanent record is not ok, we'd have to inform the experts via email, which means maintaining a de-obfuscated mailing list. This would also need an opt-out option, as some people may not want such emails. It also wouldn't reach experts that sign up after the violation has occurred, unless we send out the whole blacklist each time, again raising the question of a permanent record.
4. If we keep the details we release about offenders very sparse, it is easy for them to change their identity. If we provide a lot of detail, the above issues become even more concerning.

I suppose none of these issues is insurmountable. We could give experts the option of signing up to a mailing list after they've signed up to the volunteer list. We could then send out blacklists to this mailing list - with appropriate disclaimers to make it clear that the allegations are unverified.

All told, I'm on the fence about whether this is a good idea and how best to implement it, so I'd appreciate people's  input.

Reply:
To point 1: if we say we will remove experts from the listing as soon as a report is received about them, I think it is fair to take action to exclude participants about whom we receive reports without any further investigation.

I like the suggestion you make in point 3, to have a separate mailing list that people adding themselves to the listing could subscribe to for messages such as these.

To point 4: very sparse info is fine. You're right that people could easily get around this but I think we should not worry about such nefarious behaviour until we are actually confronted by it. To be honest, a project like this will always be vulnerable to coordinated, intentional trolling/misbehaviour. Let's just hope that that doesn't happen!

Reply:
1. For point 2. do we want to provide a mechanism for someone to contest their removal or is that too many turtles down? We would need to keep the justification public.
2. Do we want to request minimal evidence of violation from the first interaction? We may want to mention this explicitly in the CoC.

Reply:
In adapting the CoC I wanted to very specifically exclude all forms of arbitration on our part. If we allow rebuttals, we have to judge whether they are valid and credible. If we ask for "minimal evidence", we have to judge whether it is sufficient and credible.

I really don't think we're in any position to handle this sort of thing, so I'd strongly favor steering well clear of it. Let's keep our role to that of providing an address book.

As for removal from the list, the stakes of being removed are so minuscule that requiring evidence and allowing appeals may well impose a much bigger burden on everyone involved compared to just removing anyone who is being reported.

Reply:
Quick note: I've managed to set up the mailing list. Will update the site and the CoC tomorrow.